docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
     user: gameserver
     image: "mschnitzer/asa-linux-server:latest"
     environment:
-      - ASA_START_PARAMS=TheIsland_WP?listen?Port=7777?RCONPort=27020?RCONEnabled=True -WinLiveMaxPlayers=50
+      - ASA_START_PARAMS=TheIsland_WP?listen?Port=7777?RCONPort=27020?RCONEnabled=True -WinLiveMaxPlayers=50 -ClusterDirOverride="/home/gameserver/cluster-shared" #PUT clusterid=SECRET here to enable transfers
```

Comment:
So users need to specify `-clusterid=randomstring` in addition to `ClusterDirOverride` is that what your comment is saying?

Reply:
ClusterDirOverride should always be set if the shared volume is mounted by default imo

`ClusterDirOverride` is used to tell the gameserver where to save the character data on transfer / download / upload
`-clusterid=randomstring` will enable the user to transfer to other servers that use the same `randomstring`